src/DateLabel/DateLabel.stories.tsx

```diff
@@ -21,7 +21,8 @@ const Template: StoryFn<DateLabelProps> = args => {
 // Default
 export const Default = {
   args: {
-    label: "10-09-24 10:24:08"
+    label: "10-09-24 10:24:08",
+    tooltip: ""
```

Comment:
I think the default story is supposed to show all of the props functioning

```suggestion
    tooltip: "Created at"
```